configuration/pih/htmlforms/labResults.xml

```diff
@@ -159,7 +159,18 @@
                     </div>
                 </field>
             </p>
-
+            <br/>
+            <p>
+                <label for="sample-id">
+                    <uimessage code="pihcore.lab.sample_id"/>
+                </label>
+                <span class="small">
+                    <field>
+                        <obs  id="sample-id" conceptId="CIEL:162086"
+                              placeholder="Enter Sample ID"/>
```

Comment:
@mogoodrich I didn't follow the conversation today post-standup about specimen id.  It appears that the concept is the same from this file.  What changed for the data/concept?  (Team me if that's easier)

Reply:
Just talked to @lnball , and we are good. No one is using this form yet. Thanks!